.github/release-drafter.yml

```diff
@@ -55,7 +55,7 @@ autolabeler:
       - 'CHANGELOG.md'
       - 'zebra-consensus/src/checkpoint/*-checkpoints.txt'
       # Developer-only changes
-      - 'zebra-utils'
+      - 'only zebra-utils'
```

Comment:
I'm not sure if there is any way to say "only zebra-utils" in the regex:
https://github.com/release-drafter/release-drafter#autolabeler

Maybe we'll just have to delete it.

```suggestion
```

Reply:
Oh, I must've misunderstood one of your previous comments.